lasagne/layers/conv.py

```diff
@@ -281,7 +281,7 @@ def __init__(self, incoming, num_filters, filter_size, stride=1, pad=0,
         self.num_filters = num_filters
         self.filter_size = as_tuple(filter_size, n, int)
         self.flip_filters = flip_filters
-        self.stride = as_tuple(stride, n, int)
+        self.stride = as_tuple(stride, n)
```

Comment:
I see. It's not limited to Python 3, actually:
```
In [1]: isinstance(np.int32(3), int)
Out[1]: False

In [2]: isinstance(np.int64(3), int)
Out[2]: True
```
In Python 3, both are False.

---

Anyway, I think this is outside the scope of this PR. It affects a lot of existing layers and should be addressed in a separate issue. Please better adapt the test not to pass a numpy array here.